server/src/db/schemas/category.schema.ts

```diff
@@ -5,13 +5,15 @@ const Schema = mongoose.Schema;
 export interface Category {
     id?: mongoose.Types.ObjectId
     name: string,
-    imageUrl: string,
-    sentenceBeginning: string
+    sentenceBeginning: string,
+    imageUrl: string
+    items: Array<Item>
```

Comment:
Remove `items`.
Go over `main` branch and make sure the schemas are the same (you don't need to change any schema from `main`)